src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -105,7 +105,7 @@ const SelectPanel = props => {
             {slots.header || <SelectPanel.Header />}
             {childrenInBody}
             {/* render default footer as fallback */}
-            {slots.footer || <SelectPanel.Footer />}
+            {slots.footer}
```

Comment:
>I forgot that we would need to add Suspense support to useSlots first before making footer optional. 

This is probably my ignorance but what is the reason of wrapping the footer in Suspense? 🤔

Reply:
If we need to fetch data before rendering the list, we'd like the footer to also be hidden until then:


https://github.com/primer/react/assets/1863771/8ce5704a-8162-439a-a105-4ee4c55ce89e

Reply:
Ahh that makes sense!!